### PR TITLE
[PWGCF] Change output structure and implement centrality dependence

### PR DESCRIPTION
**Pull request by omassen**
Changed the structure how the histograms are stored in the output file, using folders for the A- and C-side of the ZDC. Also implemented a more realistic way of filling the histograms for the different sectors on the two sides. Finally a maximum centrality limit and centrality dependence was implemented for most histograms.

### CODE REVIEW

File: PWGCF/TwoParticleCorrelations/Tasks/neutronProtonCorrZdc.cxx [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+  Filter collisionVtxZ = nabs(aod::collision::posZ) < 10.f;
+
+  using CentralitiesRun3 = aod::CentFT0Cs;
+  using CentralitiesRun2 = aod::CentRun2V0Ms;
+  using BCsRun3 = soa::Join<aod::BCs, aod::Timestamps, aod::BcSels, aod::Run3MatchedToBCSparse>;
+
+  void init(InitContext const&)
+  {
+    // define axes you want to use
+    const AxisSpec axisCounter{4, 0, +2, ""};
```
Comment (victor-gonzalez) [inline PWGCF/TwoParticleCorrelations/Tasks/neutronProtonCorrZdc.cxx:60]: I would suggest starting the x axis in `-0.5` and run it up to the number of decided bins plus `0.5`
Then use an `enum` to define named constants for each case wanted wanted to be considered and use that named constants to fill the histogram
Also the different bins can also have its own name if so is decided

**Review by victor-gonzalez — CHANGES_REQUESTED** [review]
It seems that the version of your file in the master is diverging with respect to the base you used for your changes 

You have to rebase your changes to the latest version of the master

**Comment by victor-gonzalez** [comment]
You can see the conflicting lines following the `web editor` under the message of conflicting branch
I would suggest not resolving the conflict on the web but rebasing your changes to the master locally first

**Comment by victor-gonzalez** [comment]
Have in mind that in this PR you are modifying the `CMakeLists.txt` to include your source file, but it is already included in the master as you can check in https://github.com/AliceO2Group/O2Physics/tree/master/PWGCF/TwoParticleCorrelations/Tasks

**Comment by omassen** [comment]
I made a mistake somewhere in my git, such that I indeed included former changes in this pull request. I will resolve these issues locally, close this pull request and open a new, clean one where I have resolved the issues.

**Comment by victor-gonzalez** [comment]
Please, do not close this one. The whole history will be lost
Any update you do in your local and remote repos will update this PR accordingly and the history will be preserved

**Comment by omassen** [comment]
It seems I cannot reopen the pull request due to the way I resolved the issue (which might not have been the best way to do so), my apologies for this. However, would it also be okay if I explicitly refer back to this closed PR in the new PR?

**Comment by omassen** [comment]
Ah my apologies, I will try to reopen this PR.

**Comment by victor-gonzalez** [comment]
No worries!
Please, go ahead with the new one and refer to this one on it

**Pull request closed without merging**